attractions/modal/modal.svelte

```diff
@@ -5,6 +5,7 @@
   let _class = null;
   export { _class as class };
   export let open = false;
+  export let clickaway = true;
```

Comment:
Name is not really clear. How about `persistent` (with inverted behavior) or `closeOnClickaway`?

Reply:
`persistent` sounds even less clear, and `closeOnClickaway` is too long and reminiscent of an event. The clickaway term is somewhat known + we have docs to clarify the meaning, so I don't think a change is worth it